src/tests/ftest/nvme/enospace.py

```diff
@@ -206,12 +220,14 @@ def test_enospace_lazy_with_bg(self):
 
         :avocado: tags=all,full_regression
         :avocado: tags=hw,medium,ib2
```

Comment:
Pre-existing - if you have to push again: `ib2` is no longer used. here and other methods below
```suggestion
        :avocado: tags=hw,medium
```